frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx

```diff
@@ -37,13 +31,18 @@ const EventSinkServicesOverviewList: React.FC<EventSinkServicesOverviewListProps
   const { name: sinkName, kind: sinkKind, apiVersion: sinkApiversion } =
     spec?.sink?.ref || spec?.sink || {};
   const sinkUri = status?.sinkUri;
-  const deploymentData = current?.obj?.metadata?.ownerReferences?.[0];
   const apiGroup = apiVersion.split('/')[0];
   const linkUrl = `/search/ns/${namespace}?kind=${PodModel.kind}&q=${encodeURIComponent(
     `${apiGroup}/${_.lowerFirst(kind)}=${name}`,
   )}`;
   const { group, version } = (sinkApiversion && groupVersionFor(sinkApiversion)) || {};
   const isSinkReference = !!(sinkKind && sinkName && group && version);
+  const { podData } = usePodsWatcher(obj, obj.kind, namespace);
+  const deploymentData = React.useMemo(
+    () => podData?.current?.obj?.metadata?.ownerReferences?.[0],
+    [podData],
+  );
```

Comment:
Why would we need `useMemo` here instead of:
```suggestion
  const deploymentData = podData?.current?.obj?.metadata?.ownerReferences?.[0];
```